test/gleam/Dockerfile

```diff
@@ -0,0 +1,76 @@
+#--------------------------------------
+# Image: base
+#--------------------------------------
+FROM ubuntu:focal as base
+
+RUN touch /.dummy
+
+ARG APT_HTTP_PROXY
+ARG CONTAINERBASE_DEBUG
+
+ENV BASH_ENV=/usr/local/etc/env
+SHELL ["/bin/bash" , "-c"]
+
+COPY src/ /
+
+RUN install-containerbase
+
+COPY --chown=1000:0 test/gleam/test test
+WORKDIR /test
+
+RUN prepare-tool gleam
+
+#--------------------------------------
+# build
+#--------------------------------------
+FROM base as build
+
+# renovate: datasource=docker versioning=docker
+RUN install-tool gleam 0.34.1
+
+# renovate: datasource=docker versioning=docker
+RUN install-tool gleam 0.34.1
+
+
+USER 1000
+
+#--------------------------------------
+# test a: root
+#--------------------------------------
+FROM build as testa
+
+RUN set -ex; \
+    cd a; \
+    cargo update; \
```

Comment:
Cargo mentions left over from copy paste?

Reply:
Yeeeup. Will fix that up tomorrow